src/events/vote_persona.js

```diff
@@ -7,6 +7,25 @@ const Redis = require('../server/redis.service');
 
 const debug = require('debug')('have_you_heard');
 
+// TODO - modularize personas and multiplex with other languages
+const pt_personas = [
+    'Antivacina',
+    'Bonosaro',
+    'Eron Must',
+    'Lulo',
+    'Salvio',
+    'Tia do Zap',
+    'Tump',
+    'Vegana'
```

Comment:
In the future we could modularize this and change from actual persona names to just  index numbers 0~8. And then the actual persona is deduced by the client from Language and index.

Reply:
Makes sense. For the back-end the personas are meaningless.